docs/site/downloads/cldr-48.md

```diff
@@ -7,6 +7,7 @@ title: CLDR 48 Release Note
 | No. |    Date    | Rel. Note |  Data  |  Charts  | Spec |   Delta  | GitHub Tag | Delta DTD | CLDR JSON |
 |:---:|:----------:|:---------:|:------:|:--------:|:------------:|:---:|:----------:|:---------:|:---------:|
 |  48 | 2025-10-29 | [v48](/index/downloads/cldr-48) | [CLDR48](https://unicode.org/Public/cldr/48/) | [Charts48](https://unicode.org/cldr/charts/48/) | [LDML48](https://www.unicode.org/reports/tr35/48/tr35.html) | [Δ48](https://unicode-org.atlassian.net/issues/?jql=project%20%3D%20CLDR%20AND%20status%20%3D%20Done%20AND%20resolution%20%3D%20Fixed%20AND%20fixversion%20%3D%2048%20ORDER%20BY%20priority%20DESC) | [release-48](https://github.com/unicode-org/cldr/releases/tag/release-48) | [ΔDtd48](https://www.unicode.org/cldr/charts/48/supplemental/dtd_deltas.html) | [48.0.0](https://github.com/unicode-org/cldr-json/releases/tag/48.0.0) |
+|  48.1 | TBD | [v48](/index/downloads/cldr-48) | TBD | TBD | TBD | [Δ48.1]([https://unicode-org.atlassian.net/issues/?jql=project%20%3D%20CLDR%20AND%20status%20%3D%20Done%20AND%20resolution%20%3D%20Fixed%20AND%20fixversion%20%3D%2048%20ORDER%20BY%20priority%20DESC](https://unicode-org.atlassian.net/issues/?jql=project%20%3D%20CLDR%20AND%20status%20%3D%20Done%20AND%20resolution%20%3D%20Fixed%20AND%20fixversion%20%3D%2048.1%20ORDER%20BY%20priority%20DESC)) | TBD | TBD | TBD |
```

Comment:
- Instead of [v48] we should have [v48.1] and a link to an anchor within the cldr-48 download page with into about 48.1
- For Data we should just have "n/a", we do not usually provide zip files for dot releases
- For Spec we should link to tr35/proposed.html and Steven should update the redirects so that works
- The [Δ48.1] link=k seems strange. Why not just change the fixVersion from the [Δ48] link?